src/controllers/dynakube/version/healthcheck.go

```diff
@@ -0,0 +1,79 @@
+package version
+
+import (
+	"context"
+	"time"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/src/registry"
+	containerv1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/pkg/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	DefaultHealthConfigInterval    = time.Second * 10
```

Comment:
```suggestion
	DefaultHealthConfigInterval    = 10 * time.Second
```
Usually we use <num> than seconds.

Reply:
Fixed